PosterCalibrationSample/ReadMe.md

```diff
@@ -0,0 +1,35 @@
+## PosterCalibrationSample
+
+This sample uses an OpenCV plugin to find a specified poster texture in the PV cam view and aligns
+a transform to that poster.  Other objects can be anchored to that transform to align them to a known real-world location.
+The poster will be aligned to within a couple of centimeters, and then can use a manual alignment mechanism if finer alignment is needed.
```

Comment:
Does the accuracy of the alignment correspond to the size of the poster being aligned?